hera_xblocks/hera_pages/hera_pages/static/js/src/hera_pages.js

```diff
@@ -1,36 +1,26 @@
 /* Javascript for HeraPagesXBlock. */
 function HeraPagesXBlock(runtime, element) {
     var isFormValid;
-    $('#slider-popup').popup();
-    $('.slidebar-wrapper, .image-wrapper').slick({
-        slidesToShow: 1,
-        slidesToScroll: 1,
-        fade: true,
-        adaptiveHeight: true,
-        infinite: false
-        // asNavFor: '.slider-nav'
-      });
-
     let shouldShowMessage = true;
 
-    $(".button-prev").click((e) => {
-        var imgSlideId = $('.image-wrapper').slick('slickCurrentSlide');
-        var contentSlideId = $('.slidebar-wrapper').slick('slickCurrentSlide');
+    $(".button-prev", element).click((e) => {
```

Comment:
please replace arrow functions to the basic `function(){}`, not all browsers support it (like edge)